src/components/vertical-menu/vertical-menu-trigger.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { PaddingProps } from "styled-system";
+import tagComponent, { TagProps } from "../../__internal__/utils/helpers/tags";
+
+import { filterStyledSystemPaddingProps } from "../../style/utils";
+import { StyledVerticalMenuItem, StyledTitle } from "./vertical-menu.style";
+
+export interface VerticalMenuTriggerProps extends PaddingProps, TagProps {
+  /** Height of the menu trigger */
+  height?: string;
+  /** Title of the menu trigger */
+  children: string;
+  /** Callback passed to the menu trigger */
+  onClick: (ev: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+export const VerticalMenuTrigger = ({
+  height = "40px",
+  p = 2,
+  onClick,
+  children,
+  ...rest
+}: VerticalMenuTriggerProps) => {
+  const paddingProps = filterStyledSystemPaddingProps(rest);
+
+  return (
+    <StyledVerticalMenuItem
```

Comment:
I've tried to figure out what the cause is, but currently it seems `VerticalMenuTrigger` cannot be navigated to with the keyboard despite having `tabIndex={0}`
(open storybook locally, go to story `full screen` and make the viewport width smaller - the menu trigger does not seem to be focused when tab is pressed)

Reply:
I believe it's the FocusTrap or something within the VerticalMenuFullScreen that is causing as it is always there but hidden etc. For now I am just going to return null is isOpen is false and we can investigate when time pressures aren't there